code/API_definitions/qos-profiles.yaml

```diff
@@ -206,6 +206,14 @@ components:
           example: "QoS profile for video streaming"
         status:
           $ref: "#/components/schemas/QosProfileStatusEnum"
+        countryAvailability:
+          description: |
+            A list of countries, and optionally networks, for which the API provider makes the profile available.
+            - When profiles are filtered for a specific device (i.e. using a 3-legged access token or an explicit device filter parameter), the profile can be used by that device in any of these countries when connected to any of the listed networks, or the API provider's network if none are listed
```

Comment:
We may need to clarify that the service will work in roaming mode for this device in the listed countries/networks outside of the home network.